ui/templates/base.html

```diff
@@ -15,6 +15,15 @@
     <script type="text/javascript">
     var SETTINGS = {{ js_settings_json|safe }};
     var CURRENT_PAGE_URL = '{{ request.build_absolute_uri }}';
+    {% if SETTINGS.gtmID %}
```

Comment:
I think this needs to be checking the key in `js_settings_json` because `SETTINGS` is a JS var here